app/javascript/components/nav/components/menu-entry/component.jsx

```diff
@@ -58,7 +58,7 @@ const Component = ({ closeDrawer, menuEntry, mobileDisplay, jewelCount, username
   const navItemName = name === "username" ? username : i18n.t(name);
 
   const renderNavAction = (
-    <li>
+    <li style={{display: show === true ? "block" : "none"}}>
```

Comment:
Instead of hiding the component with CSS, let's skip rendering it altogether. I think that could be done in the main navigation component.

Reply:
Done!